### DISCUSSION TITLE
Unable to use external HTTP requests after installing Dify with docker-compose on Ubuntu

### DISCUSSION THREAD

**Discussion by StanleyChanH** [langgenius/dify#8423] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I installed Dify using docker-compose on Ubuntu, and the service is running normally. However, all HTTP requests to external nodes are failing. For example, I cannot use DuckDuckGo for translations. None of the configuration files have been modified.

**Comment by wydevise** [comment]
The same problem, I am on a Mac and cannot access HTTP on the Windows LAN
